Cargo.toml

```diff
@@ -163,3 +163,49 @@ workspace = true
 [[example]]
 name = "kerberos"
 required-features = ["network_client"]
+
+[profile.ffi-production]
+inherits = "release"
+strip = "symbols"
+lto = true
+opt-level = "z"
+codegen-units = 1
+panic = "abort"
```

Comment:
Also set `panic = "abort"` because the code is typically written in a panic-resistant way I think. Does that sound good to you? @TheBestTvarynka